calendarsubscription/store.go

```diff
@@ -0,0 +1,152 @@
+package calendarsubscription
+
+import (
+	"context"
+	"database/sql"
+	uuid "github.com/satori/go.uuid"
+	"github.com/target/goalert/permission"
+	"github.com/target/goalert/util"
+	"github.com/target/goalert/util/sqlutil"
+	"github.com/target/goalert/validation/validate"
+	"time"
+)
+
+type Store interface {
```

Comment:
I think we were going to stop using the giant interfaces for these and just use the store as a struct going forward like in `auth/basic`, `config` and `timezone`.